force-app/main/default/lwc/errorSettings/errorSettings.js

```diff
@@ -0,0 +1,59 @@
+import { LightningElement } from "lwc";
+
+import stgSystemSettingsNav from "@salesforce/label/c.stgSystemSettingsNav";
+import stgSystemSettingsTitle from "@salesforce/label/c.stgSystemSettingsTitle";
+import stgErrorSettingsTitle from "@salesforce/label/c.stgErrorSettingsTitle";
+
+export default class ErrorSettings extends LightningElement {
+    isEditMode = false;
+    affordancesDisabledToggle = false;
+
+    labelReference = {
+        systemSettingsPageHeading: stgSystemSettingsNav,
+        systemSettingsTitle: stgSystemSettingsTitle,
+        errorSettingsTitle: stgErrorSettingsTitle,
+    };
+
+    get affordancesDisabled() {
+        if (!this.isEditMode || this.affordancesDisabledToggle === true) {
+            return true;
+        }
+        return undefined;
+    }
+
+    handleSettingsEditModeChange(event) {
+        this.isEditMode = !event.detail;
+        this.affordancesDisabledToggle = event.detail;
+    }
+
+    handleSettingsSaving(event) {
+        this.affordancesDisabledToggle = true;
+        // TODO: perform client side validation
+
+        // if validation fails, call this.handleValidationFailure()
+        //this.template.querySelector("c-settings-save-canvas").handleValidationFailure();
+
+        // else, update hierarchy settings
+        this.template.querySelector("c-settings-save-canvas").updateHierarchySettings();
+    }
+
+    handleSettingsSaveCompleted(event) {
+        this.affordancesDisabledToggle = false;
+        this.refreshAllApex();
+    }
+
+    refreshAllApex() {
+        // Promise.all([
```

Comment:
Same to this one. I imagine this is because you want to update this in another PR?

Reply:
yes, that is correct :). It will be uncommented for the next PRs.